Content.Server/Players/PlayTimeTracking/PlayTimeTrackingSystem.cs

```diff
@@ -58,6 +62,11 @@ public override void Shutdown()
 
     private void CalcTrackers(ICommonSession player, HashSet<string> trackers)
     {
+        if (_adminManager.IsAdmin(player))
+        {
+            trackers.Add(PlayTimeTrackingShared.TrackerAdmin);
+        }
+
         if (_afk.IsAfk(player))
```

Comment:
Is it a good idea to bump admin playtime above overall. If you really want to bypass it for afk seems like you should do it at an admin level.

Reply:
I'm not really sure what you mean by this one sorry, I'll ping you tomorrow morning when I'm not half asleep for more info. 😴 
Thank you for the reviews though appreciate it!

Reply:
Admin playtime gets bumped before afk does which means admin time can be higher than their playtime overall which I'm not a big fan of.